src/ray/raylet/node_manager.cc

```diff
@@ -2957,6 +2968,13 @@ void NodeManager::ProcessSubscribePlasmaReady(
 
   auto message = flatbuffers::GetRoot<protocol::SubscribePlasmaReady>(message_data);
   ObjectID id = from_flatbuf<ObjectID>(*message->object_id());
+
+  // Tell task dependency manager to pull the objects.
+  // NOTE(simon): This doesn't handle cancellation and edge case invovling eviciton.
```

Comment:
Could you maybe expand on this TODO/NOTE?

Reply:
Added, PTAL!